src/daily_output.ts

```diff
@@ -286,19 +268,20 @@ export class DailyOutput {
 		if (isMetadataTransaction) {
```

Comment:
Above this if check is: 

```ts
	// track unbundled bundles
		if (bundledIn) {
			if (this.unbundledBundleTxIDs.indexOf(bundledIn) === -1) {
				this.unbundledBundleTxIDs.push(bundledIn);
			}
		}
```

Do we need to do this anymore with the new approach?

Reply:
Nice catch! 85cc6a318bf26486cb0de5e50a1fdf635fda907d